modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/shared/components/filter/hooks/useFilterFetch.es.js

```diff
@@ -47,12 +48,16 @@ const useFilterFetch = ({
 
 		const mergedItems = mergeItemsArray(staticItems, data.items);
 
-		const mappedItems = buildFilterItems({
+		let mappedItems = buildFilterItems({
 			items: mergedItems,
 			propertyKey,
 			selectedKeys,
 		});
 
+		if (formatItem) {
```

Comment:
Hey @naraandrade, What do you think if we change the `buildFilterItems` adding a parameter called `propertyLabel` (or something like this) and use this to add the required label property into the filter item? 
With this we avoid another map operation over the filter items.

Reply:
Another option is to pass the `formatItem` to `buildFilterItems`. Fell free to choose one option or propose a new one. Thx

Reply:
Thanks, @rafaprax :) I thought it would be better to not introduce this change on the buildFilterItems because, in case other filters need to add specific properties too, it could inflate the function. And the way that I did, if other filters need to manipulate the items in some way, they can just pass a function to useFilterFetch. 

But I can think about a way to pass formatItem to buildFilterItems and combine the functions to avoid another map. Thank you ^^